api/libs/iucn/src/iucn-category.ts

```diff
@@ -0,0 +1,12 @@
+export enum IUCNCategory {
```

Comment:
Moved away from `api/apps/geoprocessing/src/modules/protected-areas/protected-areas.geo.entity.ts` to avoid `libs` to import anything outside `libs`